marc_to_solr/lib/authority_traject_config.rb

```diff
@@ -14,15 +14,19 @@
   provide "solr_writer.max_skipped", "50"
   provide "marc4j_reader.source_encoding", "UTF-8"
   provide "log.error_file", "./log/traject-error.log"
-  provide "allow_duplicate_values",  false
+  provide "allow_duplicate_values", false
   provide "solr_writer.commit_on_close", "true"
 end
 
 $LOAD_PATH.unshift(File.expand_path('../../', __FILE__)) # include marc_to_solr directory so local translation_maps can be loaded
 
 to_field 'id', extract_marc('001', first: true)
 
-to_field 'marc_display', serialized_marc(:format => 'xml', :binary_escape => false, :allow_oversized => true)
+to_field 'auth_001_s', extract_marc('001', first: true)
```

Comment:
I think you can remove the 'auth_001_s` field since it's identical to the `id` field.